src/main/java/org/symphonyoss/symphony/messageml/markdown/MarkdownRenderer.java

```diff
@@ -146,15 +144,6 @@ public void visit(Link a) {
     // Note: this is the correct Markdown representation of links with text. We can't do this as we'd break legacy clients.
 //    String markdown = (StringUtils.isNotBlank(a.getTitle())) ? String.format("[%s](%s)", a.getTitle().trim(), href) : href;
```

Comment:
We should remove this commented-out line, it's not worth leaving it here as we don't know if or when can we remove that. Wdyt @ldrozdz ?